openam-scripting/src/main/java/org/forgerock/openam/scripting/StandardScriptEngineManager.java

```diff
@@ -71,7 +71,7 @@ public final class StandardScriptEngineManager extends ScriptEngineManager {
      */
     @Inject
     public StandardScriptEngineManager() {
-
```

Comment:
added backward compartibility in the followin PR: https://github.com/OpenIdentityPlatform/commons/pull/62.
Allows to use [commons](https://github.com/OpenIdentityPlatform/commons) library in OpenAM without any modification.

Reply:
have to support deploy for multiple JDKs ?

Reply:
Yes, this makes sense